pkg/pgmodel/sql_reader.go

```diff
@@ -222,13 +224,13 @@ func (q *pgxQuerier) getLabelsForIds(ids []int64) (lls labels.Labels, err error)
 	return
 }
 
-func (q *pgxQuerier) fetchMissingLabels(misses []interface{}, missedIds []int64, newLabels []interface{}) error {
+func (q *pgxQuerier) fetchMissingLabels(misses []interface{}, missedIds []int64, newLabels []interface{}) (numNewLabels int, err error) {
```

Comment:
nit: should we return newLabes as []interface{} as a return to make it clear this is the thing returned? I always get confused which things are returned out of here. Also a comment may help.

Reply:
I tried that the first time I made this edit, it didn't end up working because this is the suffix of a slice whose other values are also needed. I'll look into a better fix in a latter PR.